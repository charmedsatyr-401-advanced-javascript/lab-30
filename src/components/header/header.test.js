import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from './header.js';

describe('<Header/>', () => {
  it('is alive at application start', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('header').exists()).toBeTruthy();
  });

  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
