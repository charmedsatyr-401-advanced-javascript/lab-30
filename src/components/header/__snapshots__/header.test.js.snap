// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders correctly 1`] = `
<header>
  <h1>
    City Explorer
  </h1>
  <p>
    Enter a location below to learn about the weather, events, restaurants, movies filmed there, and more!
  </p>
</header>
`;
